forge/test/mlir/test_ops.py

```diff
@@ -14,6 +14,36 @@
 from forge.tensor import to_forge_tensors, to_pt_tensors
 
 
+@pytest.mark.parametrize(
+    "shape, mode",
+    [
+        ((1, 2048, 7, 7), "nearest"),
+        ((1, 2048, 7, 7), "bilinear"),
+    ],
+)
+@pytest.mark.xfail(reason="Found Unsupported operations while lowering from TTForge to TTIR in forward graph")
+@pytest.mark.push
+def test_interpolate(shape, mode):
```

Comment:
Do you plan to add upsample and downsample test cases in the following issues? 

Let's tr to push those tests and changes in next few PRs to get these model compiling through FFE :))

Reply:
- [torch.nn.functional.upsample](https://pytorch.org/docs/stable/generated/torch.nn.functional.upsample.html) is equivalent to torch.nn.functional.interpolate. so above test is for upsample op
- None of models didn't need downsample op till now. so code related to it are left untouched and test is not added

Reply:
Great! Thanks for confirmation Kamal :))